bftengine/src/bftengine/ViewsManager.cpp

```diff
@@ -1006,7 +993,7 @@ bool ViewsManager::addPotentiallyMissingPP(PrePrepareMsg* p, SeqNum currentLastS
     ViewChangeSafetyLogic::Restriction& r = restrictionsOfPendingView[idx];
     // if we need this message
     if (prePrepareMsgsOfRestrictions[idx] == nullptr && !r.isNull && r.digest == p->digestOfRequests()) {
```

Comment:
nit: can you please refactor this line with multple braces?
it looks totally unreadble.
We want ot be able to read it fast + reduce the chance for bugs when someone edit this line.

Reply:
done